### PR TITLE
Replaced azure-pipelines-tasks-azurermdeploycommon with arm-rest-v2

### PR DESCRIPTION
**Pull request by starkmsu**
**Task name**: AzureWebAppContainerV1

**Description**: Replaced azure-pipelines-tasks-azurermdeploycommon with arm-rest-v2

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzureWebAppContainerV1/package.json [hunk 1]
```diff
@@ -21,7 +21,10 @@
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
     "@types/uuid": "^8.3.0",
-    "azure-pipelines-tasks-azurermdeploycommon": "^3.221.1",
+    "azure-devops-node-api": "11.2.0",
+    "azure-pipelines-task-lib": "^4.1.0",
```
Comment (ozanovus) [inline Tasks/AzureWebAppContainerV1/package.json:25]: Task lib 3.4.0 might be better since task has only Node10 version.

Comment (onetocny) [reply Tasks/AzureWebAppContainerV1/package.json]: Done

**Pull request merged**